UI/script.js

```diff
@@ -0,0 +1,801 @@
+window.onload = () => {
```

Comment:
It's better to declare classes and functions not in `onLoad`, and leave here only testing code